cloud/scope/cluster.go

```diff
@@ -145,6 +146,13 @@ func (s *ClusterScope) LBSpecs() []azure.LBSpec {
 	}
 }
 
+// RouteTableSpec returns the node route table
+func (s *ClusterScope) RouteTableSpec() azure.RouteTableSpec {
```

Comment:
for consistency with the other services, and because it's likely we'll need more than one route table in the near future, let's make this 
```suggestion
func (s *ClusterScope) RouteTableSpecs() []azure.RouteTableSpec {
```

And adjust the Reconcile/Delete() functions to loop over the specs